library/cwm/src/lib/cwm/widget.rb

```diff
@@ -630,4 +643,182 @@ class RichText < AbstractWidget
 
     include ValueBasedWidget
   end
+
+  # Tab widget, usefull only with {CWM::Tabs}
+  # @see tabs example for usage
+  class Tab < CustomWidget
+    attr_accessor :initial
+
+    # content of tab, can contain another widgets
```

Comment:
\# @return [Yast::Term] contents of the tab, can contain {AbstractWidget}s

Can it be a single AbstractWidget too?